mlflow/tracking/fluent.py

```diff
@@ -249,9 +347,20 @@ def log_metric(key, value, step=None):
 
     :param key: Metric name (string).
     :param value: Metric value (float). Note that some special values such as +/- Infinity may be
-                  replaced by other values depending on the store. For example, sFor example, the
-                  SQLAlchemy store replaces +/- Inf with max / min float values.
+                  replaced by other values depending on the store. For example, the
+                  SQLAlchemy store replaces +/- Infinity with max / min float values.
     :param step: Metric step (int). Defaults to zero if unspecified.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        metrics = {"mse": 2500.00, "rmse": 50.00}
+
+        # Use context manager to create a new run and log metrics
+        with mlflow.start_run(run_name="My Runs"):
+            [mlflow.log_metric(key, value) for key, value in metrics.items()]
```

Comment:
```suggestion
        with mlflow.start_run(run_name="My Runs"):
            mlflow.log_metric("mse", 2500.00)
```

same as `log_param`.

Reply:
Done for metric, param, and tag